src/Indicator.vala

```diff
@@ -111,6 +145,25 @@ public class DateTime.Indicator : Wingpanel.Indicator {
         return main_grid;
     }
 
+    // TODO: As far as maya supports it use the Dbus Activation feature to run the calendar-app.
+    public void open_maya () {
```

Comment:
Calling it Maya is a little confusing for us newbies who only know it as "Calendar" :)

Reply:
It's a copypaste from the method in Calendar.vala, adapted to open Calendar. Maya is Calendar's old internal name. A new PR from me I'll rename these. :smile: